<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>325</width>
                <height>486</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QCheckBox" name="scalable_chbx">
                    <property name="text">
                        <string>Scalable</string>
                    </property>
                    <property name="checkable">
                        <bool>true</bool>
                    </property>
                    <property name="checked">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer_3">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>10</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>100</width>
                                    <height>0</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>130</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>Face mesh</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="faceMesh_lineEdit"/>
                    </item>
                    <item>
                        <widget class="QPushButton" name="faceMeshSelection_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>30</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>&lt;</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <widget class="QLabel" name="label_2">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>100</width>
                                    <height>0</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>100</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>FACS_blendshape</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="facsBs_lineEdit"/>
                    </item>
                    <item>
                        <widget class="QPushButton" name="faceBsSelection_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>30</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>&lt;</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>10</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QLabel" name="label_3">
                    <property name="maximumSize">
                        <size>
                            <width>16777215</width>
                            <height>15</height>
                        </size>
                    </property>
                    <property name="text">
                        <string>- utils -</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="createShapes_btn">
                    <property name="text">
                        <string>create FACS shapes and blendshape node</string>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>10</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                        <widget class="QLabel" name="label_4">
                            <property name="text">
                                <string>FACS_blendshape list</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="updateBsList_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>60</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>update</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QTreeWidget" name="updateBsList_treeWidget">
                    <property name="editTriggers">
                        <set>QAbstractItemView::NoEditTriggers</set>
                    </property>
                    <property name="selectionMode">
                        <enum>QAbstractItemView::ExtendedSelection</enum>
                    </property>
                    <property name="indentation">
                        <number>0</number>
                    </property>
                    <attribute name="headerVisible">
                        <bool>false</bool>
                    </attribute>
                    <column>
                        <property name="text">
                            <string notr="true">1</string>
                        </property>
                    </column>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                        <widget class="QPushButton" name="connectSelected_btn">
                            <property name="text">
                                <string>Add/Connect</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="rmvBlendshape_btn">
                            <property name="text">
                                <string>Remove</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="extractSelected_btn">
                            <property name="text">
                                <string>Extract/Edit</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
